applications/nrf5340_audio/broadcast_source/main.c

```diff
@@ -407,7 +406,11 @@ static int ext_adv_populate(uint8_t big_index, struct broadcast_source_ext_adv_d
 		return ret;
 	}
 
-	ret = broadcast_source_ext_adv_populate(big_index, ext_adv_data,
+	bool fixed_id = !IS_ENABLED(CONFIG_BT_AUDIO_USE_BROADCAST_ID_RANDOM);
```

Comment:
Seems like quite a few "NOT" operands. Should we rather rename to "random_id"?

Reply:
I figured since the default is having random ID, the command would be for setting the fixed ID.